public/app/events/events.ctrl.js

```diff
@@ -0,0 +1,24 @@
+
+(function() {
+
+  var eventsCtrl = function($scope, $state, eventService) {
+    eventService.getEvents()
+      .then(function(events) {
+        $scope.events = events;
+      });
+
+    $scope.addEvent = function() {
+      eventService.createEvent($scope.event)
+        .then(function() {
+          $scope.events.unshift($scope.event);
```

Comment:
@qjd2413 how do I refresh the events that are already on the page without doing this?